wandb/sdk/wandb_run.py

```diff
@@ -2787,6 +2341,537 @@ def mark_preempting(self) -> None:
+                    f'{printer.emoji("star")} View project at {printer.link(project_url)}'
+                )
+                if sweep_url:
+                    printer.display(
+                        f'{printer.emoji("broom")} View sweep at {printer.link(sweep_url)}'
+                    )
+            printer.display(
+                f'{printer.emoji("rocket")} View run at {printer.link(run_url)}'
+            )
+
+            # TODO(settings) use `wandb_settings` (if self.settings.anonymous == "true":)
+            if Api().api.settings().get("anonymous") == "true":
+                printer.display(
+                    "Do NOT share these links with anyone. They can be used to claim your runs.",
+                    status="warn",
+                )
+
+    # ------------------------------------------------------------------------------
+    # FOOTER
+    # ------------------------------------------------------------------------------
+
+    @staticmethod
+    def _footer(
+        sampled_history: Optional["SampledHistoryResponse"] = None,
+        final_summary: Optional["GetSummaryResponse"] = None,
+        poll_exit_response: Optional[PollExitResponse] = None,
+        check_version: Optional["CheckVersionResponse"] = None,
+        reporter: Optional[Reporter] = None,
+        quiet: Optional[bool] = None,
+        *,
+        settings: "Settings",
+        printer: Union["PrinterTerm", "PrinterJupyter"],
+    ) -> None:
+        Run._footer_history_summary_info(
+            history=sampled_history,
+            summary=final_summary,
+            quiet=quiet,
+            settings=settings,
+            printer=printer,
+        )
+
+        Run._footer_sync_info(
+            pool_exit_response=poll_exit_response,
+            quiet=quiet,
+            settings=settings,
+            printer=printer,
+        )
+        Run._footer_log_dir_info(quiet=quiet, settings=settings, printer=printer)
+        Run._footer_version_check_info(
+            check_version=check_version, quiet=quiet, settings=settings, printer=printer
+        )
+        Run._footer_local_warn(
+            poll_exit_response=poll_exit_response,
+            quiet=quiet,
+            settings=settings,
+            printer=printer,
+        )
+        Run._footer_reporter_warn_err(
+            reporter=reporter, quiet=quiet, settings=settings, printer=printer
+        )
+
+    @staticmethod
+    def _footer_exit_status_info(
+        exit_code: Optional[int],
+        *,
+        settings: "Settings",
+        printer: Union["PrinterTerm", "PrinterJupyter"],
+    ) -> None:
+
+        if settings._silent:
+            return
+
+        status = "(success)." if not exit_code else f"(failed {exit_code})."
+        info = [
+            f"Waiting for W&B process to finish... {printer.status(status, bool(exit_code))}"
+        ]
+
+        if not settings._offline and exit_code:
+            info.append("Press ctrl-c to abort syncing.")
```

Comment:
Control-C if platform is not win else Ctrl-C lol?

Reply:
sure will add (like your attention to details 😂 )